### PR TITLE
Update region for database data source

### PR DESCRIPTION
**Pull request by emerkle826**
I missed this change for updating data sources for #60

### CODE REVIEW

**Review by jeffbanks — COMMENTED** [review]
Looks good, one question on the `region` now that it fits the same structure as `additional_keyspaces` which has a schema.TypeList, should we consider referencing this as `regions` since there can be `1..N` entries?

**Comment by emerkle826** [comment]
> Looks good, one question on the `region` now that it fits the same structure as `additional_keyspaces` which has a schema.TypeList, should we consider referencing this as `regions` since there can be `1..N` entries?

I thought about that. Doing so would require a tiny bit more work on the user's part to upgrade as they would have to change the field in their Terraform resource file from `region` to `regions`. But it would make more sense.

**Comment by emerkle826** [comment]
@jeffbanks I'm going to go ahead and change the field to `regions` too. I'll ping you again for one last review when I update the resource as well.

**Review by jeffbanks — APPROVED** [review]
Looks good!

**Comment by emerkle826** [comment]
@jeffbanks How about an additional pass, now with some release notes added.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+```sh
+   terraform apply
+```
+3. Re-add the "data" definition.
+```sh
+   data "astra_database" "my_ext_db" {
+     database_id = "b3107622-429d-45ab-a6da-0252cb091c86"
+  }
+```
+2. Apply the change
```
Comment (jeffbanks) [inline RELEASE_NOTES.md:60]: Step #4 here?

**Review by jeffbanks — APPROVED** [review]
LGTM, just the one step fix.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (emerkle826) [reply RELEASE_NOTES.md:60]: good catch, copy-paste bites me again.

**Pull request merged**